homepage/style/badge.css

```diff
@@ -0,0 +1,33 @@
+.badge-element{
+    position: relative;   
+    border: var(--white-color) 1px solid;
+    border-radius: 20%; 
+    padding: 0.5rem;
+}
+
+.badge-red{
+    background-color: var(--red-color);
+}
+
+.badge-icon {
+    border-radius: 50%;
+    border: var(--white-color) 1px solid; 
+    font-size: 1.25rem;
+    font-weight: 600;
+    padding: 1px 5px;
+    text-align: center;
+    line-height: 1;
+    position: absolute;
+    left: 100%; 
+    top: -0.1em;   
```

Comment:
here top:0 ; would do the same thing, also try to stick to one either rem or em.